packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -35,6 +35,22 @@ export interface BucketDeploymentProps {
    */
   readonly destinationKeyPrefix?: string;
 
+  /**
+   * If this is set, matching files or objects will be excluded from the command
+   *
+   * @default - No files are excluded
+   * @see https://docs.aws.amazon.com/cli/latest/reference/s3/index.html#use-of-exclude-and-include-filters
+   */
+  readonly exclude?: string
```

Comment:
The CLI allows specifying `--exclude` and `--include` multiple times, so I believe these options should be arrays:

```suggestion
  readonly exclude?: string[]
```